### PR TITLE
update c# get started for VS 2019

### PR DESCRIPTION
**Pull request by mairaw**
Contributes to #16046 

@BillWagner I thought we could create a release branch to make it easier to make the changes one article at a time. Let me know what you think.

### CODE REVIEW

File: docs/core/tutorials/debugging-with-visual-studio.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ ms.custom: "vs-dotnet, seodec18"
 ---
```
Comment (BillWagner) [inline docs/core/tutorials/debugging-with-visual-studio.md]: Should we add an `ms.custom` field for `updateeachrelease`?

File: docs/core/tutorials/index.md [hunk 2]
```diff
@@ -6,13 +6,13 @@ ms.date: 03/16/2017
 titleSuffix: ""
 ms.custom: seodec18
 ---
-# Learn .NET Core and the .NET Core SDK tools by exploring these Tutorials
+# Learn .NET Core and the .NET Core SDK tools by exploring these tutorials
 
 The following tutorials are available for learning about .NET Core.
 
-## Building applications with Visual Studio 2017
+## Create applications with Visual Studio
```
Comment (BillWagner) [inline docs/core/tutorials/index.md]: I like that this is now version independent. That's why I didn't make a comment on the metadata for this file.

File: docs/core/tutorials/with-visual-studio.md [hunk 3]
```diff
@@ -1,82 +1,93 @@
 ---
-title: Build a C# Hello World application with .NET Core in Visual Studio 2017
-description: Learn how to build a simple .NET Core console application with C# using Visual Studio 2017.
+title: Create a C# Hello World application with .NET Core in Visual Studio 2019
+description: Learn how to create your first .NET Core console application with C# using Visual Studio 2019.
 author: BillWagner
 ms.author: wiwagn
-ms.date: 09/13/2017
+ms.date: 12/03/2019
 ms.custom: "vs-dotnet, seodec18"
```
Comment (BillWagner) [inline docs/core/tutorials/with-visual-studio.md]:
```suggestion
ms.custom: "vs-dotnet, seodec18 updateeachrelease"
```

**Review by BillWagner — APPROVED** [review]
This is a great start to updating all this content @mairaw. I made some notes about updating these files to add the `updateeachrelease` metadata for these articles.

You can merge this in when you are ready.

**Comment by BillWagner** [comment]
I like your plan of using a release branch here. 

Should we make a task list on #16068 with all the articles or sections that need to be updated?

**Comment by mairaw** [comment]
Good idea. I'll start the list at #16046 and we can split.

File: docs/core/get-started.md [hunk 4]
```diff
@@ -3,7 +3,7 @@ title: Get started with .NET Core
 description: Find resources to learn how to build .NET Core applications on Windows, Linux and macOS.
 author: thraka
 ms.author: adegeo
-ms.date: 09/19/2019
+ms.date: 12/03/2019
```
[the remark this answers is not in the document]
Comment (mairaw) [reply docs/core/get-started.md]: probably the one we have for vsrelease

[the remark this answers is not in the document]
Comment (mairaw) [reply docs/core/get-started.md]: it seems the one used for that is vs-dotnet

File: docs/core/tutorials/with-visual-studio.md (hunk 3 above)
Comment (mairaw) [reply docs/core/tutorials/with-visual-studio.md]: vs-dotnet is the one being used for vs releases. But I'm happy to change that one or reconsider this suggestion if we prefer the updateeachrelease instead (or a different one)

**Pull request merged**